cfgov/jinja2/v1/_includes/molecules/info-unit.html

```diff
@@ -49,8 +49,13 @@
          change to a boolean parameter. #}
 <div class="m-info-unit
             {{ 'm-info-unit__' ~ value.modifier if value.modifier else '' }}">
-
     {% if value.image %}
+        {# TODO: This code was added for the sake of A / B testing. #}
+        {% set use_image_link = request.url.endswith( '/about-us/careers' )
```

Comment:
This doesn't work if [`APPEND_SLASH`](https://docs.djangoproject.com/en/1.8/ref/settings/#append-slash) is set to `True`, which is how we have it configured (the page is at `/about-us/careers/`). This didn't work locally for me without making that change.